client/my-sites/plugins/main.jsx

```diff
@@ -567,12 +289,15 @@ export default React.createClass( {
 		return some( this.props.sites.getSelectedOrAllWithPlugins(), site => site && site.jetpack && site.canUpdateFiles );
 	},
 
-	updateAllPlugins() {
-		PluginsActions.removePluginsNotices( this.state.notices.completed.concat( this.state.notices.errors ) );
-		this.state.plugins.forEach( plugin => {
-			plugin.sites.forEach( site => PluginsActions.updatePlugin( site, site.plugin ) );
-		} );
-		this.recordEvent( 'Clicked Update all Plugins', true );
+	showPluginListPlaceholders( isWpCom ) {
```

Comment:
I think it would make sense to move this method to PluginList and make that component just receive if the list is a .com or a jetpack one, so it can check if it have to show any placeholder itself